packages/accounts-base/accounts_server.js

```diff
@@ -1078,11 +1078,13 @@ Ap._generateStampedLoginToken = function () {
 function expirePasswordToken(accounts, oldestValidDate, tokenFilter, userId) {
   var userFilter = userId ? {_id: userId} : {};
 
-  accounts.users.update(_.extend(userFilter, tokenFilter, {
-    $or: [
-      { "services.password.reset.when": { $lt: oldestValidDate } },
-      { "services.password.reset.when": { $lt: +oldestValidDate } }
-    ]
+  accounts.users.update(_.extend(userFilter, 
+    {$and: [tokenFilter, {
+      $or: [
+        { "services.password.reset.when": { $lt: oldestValidDate } },
+        { "services.password.reset.when": { $lt: +oldestValidDate } }
+      ]
+    }]
```

Comment:
Great question, and hard to search for dots on Google. :smile:

This is `Object` spread notation, taking advantage of the [Spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_operator) but using a Stage-3 ECMAScript proposal [seen here](https://github.com/sebmarkbage/ecmascript-rest-spread).  Stage-3 proposals are very likely to become a part of the official specification and we have enabled in Meteor [via `babel-preset-meteor`](https://github.com/meteor/babel-preset-meteor/blob/42a3ad8484ea7d01c5d6b95aba9a102a715c8abc/index.js#L37).